packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -65,6 +65,13 @@
                 ></c-quantic-generated-answer-rephrase-buttons>
               </div>
             </template>
+            <template lwc:if={shouldShowDisclaimer}>
+              <slot name="disclaimer">
+                <div class={disclaimerCssClass} data-cy="generated-answer__disclaimer">
```

Comment:
Some of the styling classes present in `disclaimerCssClass ` needs to be outside the slot, otherwise all these styles will not be take in consideration when displaying custom disclaimers yielding an unwanted display.

For example:
```
<c-quantic-generated-answer engine-id={engineId} answer-style={answerStyle} multiline-footer={multilineFooter}>
    <div slot="disclaimer">Custom Disclaimer</div>
</c-quantic-generated-answer>
```

will give me this::
 
<img width="692" alt="Screenshot 2024-03-22 at 9 28 27 AM" src="https://github.com/coveo/ui-kit/assets/86681870/2dd664ae-ef8d-4903-a50e-8eddc2beb6cf">

Reply:
Yes my goal with not putting the css class outside the slot was to allow the customers to style their own disclaimer without "competing" with our own style.

I know it's easier for them if they just want to add some text, but if they would want to display the disclaimer differently it can become impossible if the classes are outside the slot. What do you think?

Reply:
Indeed I get your point, but we can at least set the class that would display the disclaimer  under the citation and the rephrase buttons outside the slot so we know for sure that the disclaimer will be positioned in the right position.
Then, all the other classes can be set inside the slot so they can easily be overwritten by the classes that the user wanna use to style the disclaimer.